docs/codeql/codeql-language-guides/codeql-for-java.rst

```diff
@@ -10,7 +10,7 @@ Experiment and learn how to write effective and efficient queries for CodeQL dat
 
 .. pull-quote:: Enabling Kotlin support
 
-   CodeQL treats Java and Kotlin as parts of the same language, so to enable Kotlin support you should enable ``java`` as a language.
+   CodeQL treats Java and Kotlin as parts of the same language, so to enable Kotlin support you should enable ``java-kotlin`` as a language.
```

Comment:
My understanding is we don't version these docs based on CodeQL version, so this is OK.